temporal-sdk/src/main/java/io/temporal/workflow/Workflow.java

```diff
@@ -1186,6 +1186,16 @@ public static void upsertTypedSearchAttributes(
     WorkflowInternal.upsertTypedSearchAttributes(searchAttributeUpdates);
   }
 
+  /**
+   * Updates a Workflow Memos by applying {@code memoUpdates} to the existing Memos set attached to
```

Comment:
Can you clarify how a memo value is removed? Is this done by setting a null object? So can memo contain a null value for a key? In .NET we accept [memo update objects](https://dotnet.temporal.io/api/Temporalio.Workflows.Workflow.html#Temporalio_Workflows_Workflow_UpsertMemo_Temporalio_Workflows_MemoUpdate___) to remove any ambiguity. I forget what Go does here.

Reply:
Yes by setting a null value same as Go.

Reply:
Is there a difference between a null value with key present and an absent key? Specifically, if I do an `upsertMemo(Collections.singletonMap("key-that-exists-from-start", null))`, what will the raw memo be if I do describe workflow?

Reply:
`key-that-exists-from-start` would be removed by the server just like Go

Reply:
:+1: May be worth mentioning in docs that that is how to remove a key